pkg/handlers/generic/lifecycle/utils/utils.go

```diff
@@ -153,40 +144,6 @@ func RetrieveValuesTemplateConfigMap(
 	return configMap, nil
 }
 
-func CreateStorageClass(
```

Comment:
Moved this out, this utils file was starting to become a kitchen sink of functions.